editor/editor_run.cpp

```diff
@@ -184,9 +184,44 @@ Error EditorRun::run(const String &p_scene) {
 		args.push_back(p_scene);
 	}
 
-	String exec = OS::get_singleton()->get_executable_path();
+	int instances = EditorSettings::get_singleton()->get_project_metadata("debug_options", "run_debug_instances", 1);
+	for (int instance_num = 1; instance_num <= instances; instance_num++) {
+		List<String> instance_args;
+		for (int i = 0; i < args.size(); i++) {
+			instance_args.push_back(args[i]);
+		}
+
+		String exec;
+		String instance_arg_setting = "editor/run/instance_" + itos(instance_num) + "_run_args";
+		String raw_custom_args = ProjectSettings::get_singleton()->get(instance_arg_setting);
```

Comment:
I feel like these should be in editor settings meta too (i.e. `EditorSettings::get_singleton()->get_project_metadata("debug_options", instance_arg_setting)`).

We don't really want to export them, they are debug/editor configurations, not something to store in the project settings.

I guess that raises the question of where to show those info, and maybe an hint to proceed towards an initial implementation of https://github.com/godotengine/godot-proposals/issues/522 ?